lib/widgets/class_image.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+
+class ClassImage extends StatelessWidget {
+  const ClassImage(
+      {@required this.backgroundColor,
+      @required this.textColor,
+      @required this.selectable,
+      @required this.selected,
+      @required this.text,
```

Comment:
The point of making this a widget was to give it a `ClassHeader` and make it figure out the colours and text internally, so we don't need to pass `classHeader.colourFromAcronym...` etc. each time we use it. Remove parameters `backgroundColor`, `textColor` and `text` and replace them with a `ClassHeader classHeader` parameter instead.

Also make `selected` false by default, so that the only required parameter is the `classHeader`.